label-application-rule-config-service-api/src/main/proto/org/hypertrace/label/application/rule/config/service/v1/label_application_rule.proto

```diff
@@ -51,6 +51,7 @@ message LabelApplicationRuleData {
       OPERATOR_UNSPECIFIED = 0;
       OPERATOR_EQUALS = 1; // operator to check if the key exists, and value is equal to the provided value
       OPERATOR_MATCHES_REGEX = 2; // operator to check if the key exists, and value matches the provided regex value
+      OPERATOR_LIES_WITHIN_IP_RANGE = 3; // operator to check if the key exists, and value lies in the provided ip range
```

Comment:
or `OPERATOR_IP_WITHIN_RANGE` ?

Reply:
Since, we want a CIDR operator, we can just call it `OPERATOR_CIDR`

Reply:
Okay